src/ansys/fluent/core/file_session.py

```diff
@@ -108,15 +109,17 @@ def add_scalar_fields_request(
 
         Raises
         ------
-        RuntimeError
+        TransactionError.SurfaceNamesIDsNotProvidedError
```

Comment:
`TransactionError` is being used to provide a namespace, but the module already provides a namespace. Also, `file_session` would addsome meaning, whereas `TransactionError` is generic.